src/main/java/telraam/database/models/Team.java

```diff
@@ -9,7 +9,7 @@ public class Team {
     private Integer id;
     private String name;
     private Integer batonId;
-    private Integer jacketNr;
+    private String jacketNr = "0";
```

Comment:
Als dat toch een default nodig heeft, dan mss '' (lege string) of 'INVALID' nemen?

Reply:
```suggestion
    private String jacketNr = "8==D";
```